### PR TITLE
Use image resizer for podcast covers

### PR DESCRIPTION
**Pull request by davidfurey**
## What does this change?

Uses the image resizer for podcast cover images on podcast articles

## Does this change need to be reproduced in dotcom-rendering ?

This will need to be reproduced when podcasts are implemented on DCR

- [ ] No
- [x] Yes (please indicate your plans for DCR Implementation)

## Screenshots
Screen Shot 2020-08-05 at 17 30 08

(changes the Orange "Full Story" image)

## What is the value of this and can you measure success?

Smaller images: less bandwidth, happier users.  For example, Full Story Australia news podcast logo (https://www.theguardian.com/australia-news/audio/2020/aug/05/how-covid-19-laid-bare-the-cracks-in-australias-aged-care-system) is currently **6.3MB**, after this change it is **7.5kB**

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by gtrufitt — APPROVED** [review]
Great!

File: applications/app/views/fragments/audioBody.scala.html [hunk 1]
```diff
@@ -53,7 +55,7 @@
             ; image <- podcast.image
             ) {
                 <div class="podcast__cover">
-                    <a href="@LinkTo {/@series.id}"><img src="@image" class="podcast__cover-image" alt></a>
+                    <a href="@LinkTo {/@series.id}"><img src="@ImgSrc(image, ImageProfile(width = Some(220)))" class="podcast__cover-image" alt></a>
```
Comment (gtrufitt) [inline applications/app/views/fragments/audioBody.scala.html:58]: I think you want to make this 440 for higher pixel density displays

Comment (davidfurey) [reply applications/app/views/fragments/audioBody.scala.html]: 👍

**Comment by mxdvl** [comment]
Hi there!

Should this PR remain open? It's been around for a while and should probably either ship or be closed.
If it needs to remain open, could you please change the base branch to main? See mention above for context.

🧹💨 @davidfurey

**Comment by jorgeazevedo** [comment]
Hi!

The `master` branch in this repository has been renamed to `main`. We’ve updated the base branch of this PR for you. So if your PR has already been approved, you can just merge as usual.

If you try to push new commits to this branch however, it might not go through due to pre-push hook failures. You’ll need to either rebase this branch on top of `main` or merge from `main` to update your hooks.

Thanks! If you have any other problems feel free to reach out on the dotcom channel.

**Review by mxdvl — APPROVED** [review]

**Pull request merged**